<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comic Collector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="IssueList" name="tableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>18</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>18</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_AddComics"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>258</width>
     <height>151</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Titles</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="TimedLineEdit" name="filterEdit"/>
           </item>
           <item>
            <widget class="SeriesList" name="comicTitles">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="showGrid">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="cornerButtonEnabled">
              <bool>false</bool>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>27</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>18</number>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>18</number>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="SeriesDetail" name="groupBox"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_AddComics">
   <property name="text">
    <string>&amp;Add Comics...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TimedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>timedlineedit.h</header>
   <slots>
    <signal>timedEdit(QString)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>SeriesList</class>
   <extends>QTableView</extends>
   <header>serieslist.h</header>
   <slots>
    <signal>seriesSelected(int)</signal>
    <slot>filterList(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>SeriesDetail</class>
   <extends>QGroupBox</extends>
   <header>seriesdetail.h</header>
   <container>1</container>
   <slots>
    <slot>setSeries(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>IssueList</class>
   <extends>QTableView</extends>
   <header>issuelist.h</header>
   <slots>
    <slot>setSeries(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterEdit</tabstop>
  <tabstop>comicTitles</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>filterEdit</sender>
   <signal>timedEdit(QString)</signal>
   <receiver>comicTitles</receiver>
   <slot>filterList(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comicTitles</sender>
   <signal>seriesSelected(int)</signal>
   <receiver>groupBox</receiver>
   <slot>setSeries(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>617</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comicTitles</sender>
   <signal>seriesSelected(int)</signal>
   <receiver>tableView</receiver>
   <slot>setSeries(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
